src/utils/capitalize-first-letter.ts

```diff
@@ -0,0 +1,10 @@
+export const capitalizeFirstLetter = (value: string) => {
```

Comment:
we use lodash and lodash-es. it already has the `titleCase` in it. Why reinvent the wheel?

Reply:
`startCase` from lodash, seems to work well and do what we need. Thanks! I removed the new method